src/test/app/PermissionedDomains_test.cpp

```diff
@@ -77,6 +78,22 @@ class PermissionedDomains_test : public beast::unit_test::suite
         env(pdomain::deleteTx(alice, domain));
     }
 
+    // Verify that PD cannot be created or updated if credentials are disabled
+    void
+    testCredentialsDisabled()
+    {
+        auto amendments = supported_amendments();
+        amendments.set(featurePermissionedDomains);
+        amendments.reset(featureCredentials);
+        testcase("Credentials disabled");
+        Account const alice("alice");
+        Env env(*this, amendments);
+        env.fund(XRP(1000), alice);
+        pdomain::Credentials credentials{{alice, "first credential"}};
+        env(pdomain::setTx(alice, credentials), ter(temDISABLED));
+        env(pdomain::deleteTx(alice, uint256(75)), ter(tecNO_ENTRY));
```

Comment:
Shouldn't this also be `temDISABLED`?

Reply:
Yes - if hypothetically PD could make sense without referring to credentials (because e.g. it would refer to something else) then I think we should be able to create it; which makes me think I ought to change the check to

```
PermissionedDomainSet::preflight(PreflightContext const& ctx)
{
    if (!ctx.rules.enabled(featurePermissionedDomains))
        return temDISABLED;

    if (ctx.tx.isFieldPresent(sfAcceptedCredentials) &&
        !ctx.rules.enabled(featureCredentials))
        return temDISABLED;

```

what do you think ? This looks a bit strange, given that `sfAcceptedCredentials` is a required field

Reply:
I think that's sort of unnecessary and if we happen to make `sfAcceptedCredentials` optional we can make that change then.